CHANGELOG

```diff
@@ -14,6 +14,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Initialize new Vue.js project in `zimui` subfolder
 - Update dependencies in pyproject.toml (pydantic, pyhumps, python-slugify)
 - Update scraper to generate JSON files for `zimui` (#212)
+- Remove template files (home.html, article.html) and `make_html_files` method in scraper.py
+- Remove locale folder and files
```

Comment:
Please be more specific with something like `Remove broken locale folder and files used for translation ; translation will be restored with #222`

Reply:
Changed in [0d7050f](https://github.com/openzim/youtube/pull/219/commits/0d7050f2ff8a5e1a5b8d17cebe5e784fe1a4cf42)